### PR TITLE
Add more testpmd options

### PR DESCRIPTION
**Pull request by atheurer**
-fix some tabs/space
-fix exit_error calls

### CODE REVIEW

File: trafficgen-server-start [hunk 1]
```diff
@@ -127,7 +155,7 @@ echo -e "${CMD_OUTPUT}"
 WORKLOAD_CPUS_SEPARATED=$(echo -e "${CMD_OUTPUT}" | grep "final cpus:" | awk '{ print $3 }' | sed -e "s/,/ /g")
 WORKLOAD_CPUS_ARRAY=(${WORKLOAD_CPUS_SEPARATED})
 
-testpmd_cores=${#WORKLOAD_CPUS_ARRAY[@]}
+testpmd_cores=$(( ${#WORKLOAD_CPUS_ARRAY[@]} - 1 ))
```
Comment (k-rister) [inline trafficgen-server-start:158]: What is this changing?

**Review by k-rister — APPROVED** [review]
Looks ok but I am curious about the one change I commented on.

**Comment by atheurer** [comment]
crap, I forgot to take that out.  I was testing with all CPUs for testpmd coming from workload_cpus and no housekeeping, as I wanted to ensure all of testpmd cpus were in the same NUMA node.

File: trafficgen-server-start (hunk 1 above)
Comment (atheurer) [reply trafficgen-server-start]: I forgot to take that out. I was testing with all CPUs for testpmd coming from workload_cpus and no housekeeping, as I wanted to ensure all of testpmd cpus were in the same NUMA node.

**Pull request merged**